python/dgl/data/movielens.py

```diff
@@ -0,0 +1,630 @@
+"""MovieLens dataset"""
+
+import os
+import re
+import zipfile
+
+import numpy as np
+import pandas as pd
+import torch as th
+from sklearn.model_selection import train_test_split
```

Comment:
I don't think `sklearn` is currently a dependency of DGL. See if you can use the utils from https://github.com/dmlc/dgl/blob/master/python/dgl/data/utils.py instead.

Reply:
the same as above.

Reply:
I didn't get it. Could you elaborate more?

Reply:
got it